Helper/Order.php

```diff
@@ -2427,6 +2427,7 @@ public function createOrderInvoice($order, $amount, $notify = false, $transactio
         $invoice->save();
 
         $order->addRelatedObject($invoice);
+        $order->setTotalPaid($amount);
```

Comment:
What will happen for partial capture case?

Reply:
This function is being called for each new capture, so it'll be called for partial capture case

Reply:
Right. Does it mean we will set the incorrect total paid for the second partial capture?